pkg/models/commandvalue.go

```diff
@@ -359,6 +359,40 @@ func (cv *CommandValue) ValueToString(encoding ...string) (str string) {
 	return
 }
 
+//TypeToString returns corresponding string representation of the ValueType.
+func (cv *CommandValue) ValueTypeToString() string {
+	switch cv.Type {
+	case Bool:
+		return "Bool"
```

Comment:
Don't use the hard code string, and please return the constant value in contract instead
https://github.com/iain-anderson/go-mod-core-contracts/blob/505d356ef65bdd8749da1347940cb4bba2b9fade/models/reading.go#L23-L35